src/components/file-input/file-input.tsx

```diff
@@ -100,9 +100,6 @@ export class FileInput {
 
     private handleClick = (event: Event) => {
         if (this.disabled) {
-            event.stopPropagation();
-            event.preventDefault();
-
```

Comment:
Is this expected behaviour? If we have the file-input component disabled that means that any click wont open the native file select dialog but will still propagate the click event.

As it was previously any click event were stopped, so if you had something like this the chip would not open a link

```tsx
<limel-file-input disabled={true}>
  <limel-chip text="cat.jpg", link={href: "/cat.jpg"} />
</limel-file-input>
```

Reply:
I would say that expected behaviour for this PR is that the component keeps working as it did before… isn't this supposed to just be a refactor?

Reply:
Yeah, the implications discussed are in <limel-file-input>. Or what do you mean?

Reply:
Ah, ok, I didn't notice that this wasn't `limel-file` 😄

I'm not sure, since I haven't yet gathered enough context, but since this component is meant to wrap some other component, I think the consumer needs to be responsible for controlling the behaviour of the wrapped component. So if they don't want anything to be clickable inside limel-file-input when limel-file-input is disabled, they have to disable the component inside it themselves. If we block all clicks, we make it impossible for the consumer to implement at more nuanced behaviour, like being able to click on existing chips, but not add new files, for example.

Reply:
Sum up of discussion we had in a huddle:

We'll pause this refactoring for now, because we don't use limel-file in the activities composer, so we don't need this refactoring right now. Then we will look into possibly changing the UI a bit. We would like there to be a simple button for browsing files to upload, rather than the currently quite weird interaction where you focus a text-field to get the file-dialog. With keyboard navigation, you currently have to tab to the input field, and then press enter to get the file-dialog. That doesn't seem very intuitive, and is probably really bad for accessibility…